en/docs/quick-start-guides/deploy-your-first-integration-as-api.md

```diff
@@ -0,0 +1,198 @@
+
+5. Under **Add New Connection**, select **HTTPS**.
+
+6. Specify the following values:
+
+    | Property            | Value                   |
+    |---------------------|-------------------------|
+    | **Connection Name** | `HelloWorldConn`        |
+    | **Base URL**        | `https://apis.wso2.com` |
+
+7. Click **Add**. You'll be directed to the **Add Get** pane again.
+
+8. Enter `/zvdz/mi-qsg/v1.0` as the **Relative Path**. So the actual endpoint is `https://apis.wso2.com/zvdz/mi-qsg/v1.0`.
+
+    <a href="{{base_path}}/assets/img/get-started/deploy-your-first-integration/add-get.png"><img src="{{base_path}}/assets/img/get-started/deploy-your-first-integration/add-get.png" alt="Add new connection" width="40%"></a>
+
+9. Click **Submit**.
+
+    Now, let's add a [Respond Mediator](https://mi.docs.wso2.com/en/latest/reference/mediators/respond-mediator/) to respond the message to the client.
+
+10. Click on the **+** icon placed just after the HTTPS GET operation to open the palette.
+
+11. Select **Respond** mediator under **Mediators**.
+
+12. Click **Add**.
+
+    <a href="{{base_path}}/assets/img/get-started/deploy-your-first-integration/design-api.gif"><img src="{{base_path}}/assets/img/get-started/deploy-your-first-integration/design-api.gif" alt="Design API" width="80%"></a>
+
+Now you have successfully designed the integration.
+
+!!! note
+    You can view the source view by clicking on the **Show Source** (`</>`) icon located in the top right corner of the VSCode.
+
+## Step 4 - Run the integration artifacts
+
+Now that you have developed an integration using the MI for the Visual Studio Code plugin, it's time to deploy the integration to the MI server runtime.
+
+Click the **Build and Run** icon located in the top right corner of VSCode.
+
+<a href="{{base_path}}/assets/img/get-started/deploy-your-first-integration/build-and-run-project.png"><img src="{{base_path}}/assets/img/get-started/deploy-your-first-integration/build-and-run-project.png" alt="Build and run" width="80%"></a>
+
+## Step 5 - Test the integration service locally
+
+Now, let's test the integration service. For that, you can use the inbuilt try-it functionality in the MI for the VSCode extension.
+
+When you run the integration artifact as in [Step 4](#step-4-run-the-integration-artifacts), the **Runtime Services** interface opens. You can see all the available services.
+
+Select `HelloWorldAPI`, which you have developed, and test the resource.
+
+<a href="{{base_path}}/assets/img/get-started/deploy-your-first-integration/test-api.gif"><img src="{{base_path}}/assets/img/get-started/deploy-your-first-integration/test-api.gif" alt="Test API" width="80%"></a>
+    
+!!! note
+    For more information, refer to [Micro Integrator Documentation](https://mi.docs.wso2.com/en/latest/).
+
+## Step 6: Push to Devant <!-- Todo Update this if this is supported by the Ballerina Integrator extension itself -->
```

Comment:
Can you create a issue for this. Hopefully this will be fixed with tonight's release

Reply:
We already have issue to address Todos in this doc